src/lib/inline-images/InlineImages.tsx

```diff
@@ -28,22 +30,23 @@ const InlineImages = (props: InlineImagesPropType): JSX.Element => {
     <div className={classes.imagesContainer}>
       {data?.map((user: AvatarDataType, index: number) => (
         <div
-          key={user && user.name ?`${user.name}-${index}`: index}
+          key={user && user.name ? `${user.name}-${index}` : index}
           className={classes.eachImage}
           style={{
             left: `${index * spaceBetweenPics}px`
           }}
           id="inline-images"
         >
           <Avatar
-            avatarUrl={user.avatarUrl}
+            avatarUrl={user.avatarUrl || defaultAvatarImage}
```

Comment:
Why do we need to pass defaultAvatarImage here? It is already passed as another prop